BookStore/src/main/java/ro/utcn/fulgadan/BookStore/business/UserService.java

```diff
@@ -0,0 +1,39 @@
+package ro.utcn.fulgadan.BookStore.business;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.utcn.fulgadan.BookStore.data.entity.User;
+import ro.utcn.fulgadan.BookStore.data.repository.UserJpaRepository;
+
+@Service
+public class UserService {
+    @Autowired
+    UserJpaRepository userJpaRepository;
+
+    //return 0 for admin, 1 for user and 2 for not found
+    public Integer checkCredentials(String username, String password) {
```

Comment:
Would have been better to use an enum for this result (instead of returning an integer and needing to resort to a comment to explain what the values mean). Also, not found could be an exception...